libslide/src/math/mod.rs

```diff
@@ -1,5 +1,7 @@
 mod gcd;
 pub use gcd::*;
 
-mod gcd_poly_zz;
-pub use gcd_poly_zz::*;
+mod poly;
+
+//mod gcd_poly_zz;
+//pub use gcd_poly_zz::*;
```

Comment:
why, aren't you benchmarking this

Reply:
I will lol. It’s not fixed yet. Won’t even compile it’s still a large wip